exporter_test.go

```diff
@@ -1877,3 +1877,227 @@ func TestParseSentinelMasterString(t *testing.T) {
+		{"2/3 okay sentinel(missing flags)", []interface{}{[]interface{}{[]byte("name"), []byte("284bc2ef46881bd71e81610152cb96031d211d28"), []byte("ip"), []byte("172.17.0.8"), []byte("port"), []byte("26379"), []byte("runid"), []byte("284bc2ef46881bd71e81610152cb96031d211d28"), []byte("flags"), []byte("sentinel"), []byte("link-pending-commands"), []byte("38"), []byte("link-refcount"), []byte("1"), []byte("last-ping-sent"), []byte("11828891"), []byte("last-ok-ping-reply"), []byte("11829539"), []byte("last-ping-reply"), []byte("11829539"), []byte("s-down-time"), []byte("11823816"), []byte("down-after-milliseconds"), []byte("5000"), []byte("last-hello-message"), []byte("11829434"), []byte("voted-leader"), []byte("?"), []byte("voted-leader-epoch"), []byte("0")}, []interface{}{[]byte("name"), []byte("c3ab3cdcaeb193bb49b16d4d3da88def984ab3bf"), []byte("ip"), []byte("172.17.0.7"), []byte("port"), []byte("26379"), []byte("runid"), []byte("c3ab3cdcaeb193bb49b16d4d3da88def984ab3bf"), []byte("link-pending-commands"), []byte("38"), []byte("link-refcount"), []byte("1"), []byte("last-ping-sent"), []byte("11828891"), []byte("last-ok-ping-reply"), []byte("11829539"), []byte("last-ping-reply"), []byte("11829539"), []byte("s-down-time"), []byte("11823815"), []byte("down-after-milliseconds"), []byte("5000"), []byte("last-hello-message"), []byte("11829434"), []byte("voted-leader"), []byte("?"), []byte("voted-leader-epoch"), []byte("0")}}, []string{"mymaster", "172.17.0.7:26379"}, twoOkSentinelExpectedMetricValue},
+	}
+	for _, tst := range tsts {
+		t.Run(tst.name, func(t *testing.T) {
+			chM := make(chan prometheus.Metric)
+			go func() {
+				e.processSentinelSentinels(chM, tst.sentinelDetails, tst.labels...)
+				close(chM)
+			}()
+			want := map[string]bool{
+				"sentinel_master_ok_sentinels": false,
+			}
+
+			for m := range chM {
+				for k := range want {
+					if strings.Contains(m.Desc().String(), k) {
+						want[k] = true
+						got := &dto.Metric{}
+						m.Write(got)
+
+						val := got.GetGauge().GetValue()
+						if int(val) != tst.expectedMetricValue[k] {
+							t.Errorf("Expected Metric value:%d didn't match to reported value:%d for test:%s  ", tst.expectedMetricValue[k], int(val), tst.name)
```

Comment:
```suggestion
							t.Errorf("Expected metric value %d didn't match to reported value %d for test %s", tst.expectedMetricValue[k], int(val), tst.name)
```

Reply:
This was updated along with other review suggestion. Thanks. :)